src/lib/PriorityOrderLib.sol

```diff
@@ -24,18 +29,33 @@ struct PriorityOutput {
 struct PriorityOrder {
     // generic order information
     OrderInfo info;
+    // The address which must cosign the full order
+    address cosigner;
     // the block at which the order becomes active
-    uint256 startBlock;
+    uint256 auctionStartBlock;
```

Comment:
based on your diagram, the order is not active here yet but rather this is just the min that the cosigner can set it to right? maybe better name is `minTargetBlock`

Reply:
changed in latest commit